reliability-engineering/terraform/modules/concourse-web/lb.tf

```diff
@@ -41,7 +41,7 @@ resource "aws_lb_listener" "concourse_web_http" {
 
     redirect {
       port        = "443"
-      protocol    = "HTTPS"
+      protocol    = "TLS"
```

Comment:
Is this valid? Looking at https://www.terraform.io/docs/providers/aws/r/lb_listener.html I'm unconvinced

Reply:
I have replaced TLS with HTTPS for redirect only. I do not understand why Terrform supports TLS for protocol for aws_lb_listener but not for redirect.

Reply:
Looking at https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticloadbalancingv2-listener-redirectconfig.html it seems to be about redirecting HTTP traffic around, so this is logical.

Reply:
Ok.